test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
+    Given Minishift has state "Running"
+     When executing "oc login --username=developer --password=developer" retrying 24 times with wait period of 5 seconds
+     Then stderr should be empty
+      And exitcode should equal 0
+      And stdout should contain
+      """
+      Login successful
+      """
+    Then executing "oc new-project rubys2i" retrying 3 times with wait period of 5 seconds
+     And stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     Now using project
+     """
+    Then executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" retrying 3 times with wait period of 5 seconds
+     And stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     Success
+     """
+    Then executing "oc logs -f bc/ruby-ex" retrying 18 times with wait period of 5 seconds
```

Comment:
This is a better solution of waiting for deployment endpoint status rather than retrying on ``` oc logs -f bc/ruby-ex``` multiple times. 
```
$ oc rollout status deploymentconfig ruby-ex --watch
Deployment config "ruby-ex" waiting on image update
Waiting for latest deployment config spec to be observed by the controller loop...
Waiting for rollout to finish: 0 out of 1 new replicas have been updated...
Waiting for rollout to finish: 0 out of 1 new replicas have been updated...
Waiting for rollout to finish: 0 out of 1 new replicas have been updated...
Waiting for rollout to finish: 0 of 1 updated replicas are available...
Waiting for latest deployment config spec to be observed by the controller loop...
Waiting for latest deployment config spec to be observed by the controller loop...
replication controller "ruby-ex-1" successfully rolled out
```